static_src/components/breadcrumbs.jsx

```diff
@@ -51,7 +51,7 @@ export default class Breadcrumbs extends React.Component {
 
     // Add the parent states
     if (org && org.name) {
-      breadcrumbs.push(<BreadcrumbsItem key="home" url="/#/">overview</BreadcrumbsItem>);
+      breadcrumbs.push(<BreadcrumbsItem key="home" url="/#/">Overview</BreadcrumbsItem>);
```

Comment:
Hm, I found lowercase a teeny bit confusing while browsing around as a user - to me, "Overview" is part of the UI, a distinct thing from orgs/spaces/apps, so it feels right for it to be consistent with the other place I saw this term, rather than consistent with org/space/app names. Defer to you all though!